### ISSUE TITLE
Visual issues in API pages

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#359]
Right now there are some problems with the API pages. Namely:
1. the DAF page is completely broken
2. the INGV one has some problems, see below:
![Schermata 2019-07-10 alle 16 04 28](https://user-images.githubusercontent.com/25111463/60979054-57e79d80-a332-11e9-8e24-6a391ad8711b.png)
3. inside the swagger specs visualization I believe there are some spacing issues:
![Schermata 2019-07-10 alle 15 31 08](https://user-images.githubusercontent.com/25111463/60979279-b44abd00-a332-11e9-87a7-6be1da5bad46.png)

## Suggestions
1. We should remove it while fixing it. 
2. In the previous version of the website (1.0) it was working as expected. It could be a templating/CSS issue.
3. I think this is a problem with the td spacing

**Comment by libremente** [comment]
Point *.2* seems fixed now.

**Comment by libremente** [comment]
The DAF page is not rendered due to a React error. 
```
TypeError: "w is undefined"
    value http://localhost:4000/assets/swagger/swagger.995f75f4.js:1924
    value http://localhost:4000/assets/swagger/swagger.995f75f4.js:1924
    _renderValidatedComponentWithoutOwnerOrContext http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _renderValidatedComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _updateRenderedComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _performComponentUpdate http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1259
    _reconcilerUpdateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    _updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    _updateDOMChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1259
    _reconcilerUpdateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    _updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    updateChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1264
    _updateDOMChildren http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1274
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    _updateRenderedComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _performComponentUpdate http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    _updateRenderedComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _performComponentUpdate http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    receiveComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    _updateRenderedComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    _performComponentUpdate http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    updateComponent http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    performUpdateIfNecessary http://localhost:4000/assets/swagger/swagger.995f75f4.js:1244
    performUpdateIfNecessary http://localhost:4000/assets/swagger/swagger.995f75f4.js:1138
    v http://localhost:4000/assets/swagger/swagger.995f75f4.js:1142
    perform http://localhost:4000/assets/swagger/swagger.995f75f4.js:1140
    perform http://localhost:4000/assets/swagger/swagger.995f75f4.js:1140
    perform http://localhost:4000/assets/swagger/swagger.995f75f4.js:1142
    y http://localhost:4000/assets/swagger/swagger.995f75f4.js:1142
    closeAll http://localhost:4000/assets/swagger/swagger.995f75f4.js:1140
    perform http://localhost:4000/assets/swagger/swagger.995f75f4.js:1140
    batchedUpdates http://localhost:4000/assets/swagger/swagger.995f75f4.js:1282
    q http://localhost:4000/assets/swagger/swagger.995f75f4.js:1142
    i http://localhost:4000/assets/swagger/swagger.995f75f4.js:1266
    enqueueSetState http://localhost:4000/assets/swagger/swagger.995f75f4.js:1266
    setState http://localhost:4000/assets/swagger/swagger.995f75f4.js:20
    handleChange http://localhost:4000/assets/swagger/swagger.995f75f4.js:1440
    h http://localhost:4000/assets/swagger/swagger.995f75f4.js:1418
    systemThunkMiddleware http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    t http://localhost:4000/assets/swagger/swagger.995f75f4.js:1424
    updateJsonSpec http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    u http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    updateJsonSpec http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    u http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    updateJsonSpec http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    u http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    parseToJson http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    systemThunkMiddleware http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    t http://localhost:4000/assets/swagger/swagger.995f75f4.js:1424
    updateSpec http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    u http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    updateSpec http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    u http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    r http://localhost:4000/assets/swagger/swagger.995f75f4.js:1920
    c http://localhost:4000/assets/swagger/swagger.995f75f4.js:504
    S http://localhost:4000/assets/swagger/swagger.995f75f4.js:504
    v http://localhost:4000/assets/swagger/swagger.995f75f4.js:487
swagger.995f75f4.js:1920:279667
```
Maybe @sebbalex we could dive into this together.

**Comment by ioggstream** [comment]
@libremente we could simplify the yaml and check if there's some parsing issue out there.

**Comment by libremente** [comment]
> @libremente we could simplify the yaml and check if there's some parsing issue out there.

@ioggstream first draft response: the problem lies in the `x-project: daf` value. Removing it makes the whole project compile again and the visualization is not broken as a result.
As such, I digged a bit in the code and I found that the real problem lies in the way such value is treated, in particular the fact that in `Hero.js` there was a check against a list which was not correctly imported. 
Importing it results in this rendered page:
![Schermata 2019-08-21 alle 17 36 45](https://user-images.githubusercontent.com/25111463/63446315-4b2e8d00-c43a-11e9-8ef4-0e5b6279ac0c.png)
 
with the correct swagger-ui loaded below. 
I guess this is it, see the PR here: #382 
At the end it was just a missing import but it was quite hard to guess, even because it's the first API with an `x-project` belonging to one of our projects (https://github.com/italia/developers.italia.it/blob/master/swagger/projects.js)
cc @sebbalex
